src/Integration.UnitTests/Binding/CSharpVBBindingConfigProviderTests.cs

```diff
@@ -170,12 +170,12 @@ public async Task GetConfig_ReturnsCorrectRuleset()
             var response = await testSubject.GetConfigurationAsync(validQualityProfile, Language.VBNET, builder.BindingConfiguration, CancellationToken.None);
 
             response.Should().BeAssignableTo<ICSharpVBBindingConfig>();
-            
+
             var actualRuleset = ((ICSharpVBBindingConfig)response).RuleSet;
             actualRuleset.Path.Should().Be(expectedRulesetFilePath);
             actualRuleset.Content.Description.Should().Be(validRuleSet.Description);
-            actualRuleset.Content.Rules.Should().AllBeEquivalentTo(validRuleSet.Rules);
-            actualRuleset.Content.ToolsVersion.ToString().Should().Be(validRuleSet.ToolsVersion);
+            actualRuleset.Content.Rules.Should().BeEquivalentTo(validRuleSet.Rules);
```

Comment:
The VS `RuleSet.Rules` and Core `RuleSet.Rules` are different types of collection so I had to use a different FluentAssertion.